### ISSUE TITLE
Bug: Jarring volume normalization effect limits max volume on 16 Pro Max

### ISSUE THREAD

**Issue by mapleroyal** [TortugaPower/BookPlayer#1206]
## Environment

 - OS Version: [ios 18.0.1]
 - App Version: [5.4.1-74]

## Description

On the 16 pro max specifically, increasing the volume past about 70% introduces an effect I can only describe as a really bad volume normalization algorithm that sounds like it’s forcing output down to 70% max volume. I’ve returned one device for a replacement and the issue persists.

It’s jarring and extremely obnoxious given how frequently there are pauses in spoken content, in addition to being frustrating as it robs me of the loudest 30% of the volume range.

## Steps to reproduce the behavior
1. Play an audiobook 
2. Increase volume to 70%
3. Increase volume past 70% and notice that it doesn’t get any louder—except it does, for a split second after every pause between speaking, then it’s immediately reduced down to 70% until another gap in the spoken content.

**Comment by GianniCarlo** [comment]
Is this happening with AirPods? I had a similar experience with a podcast app, and it was due to the AirPod settings of Personalized Volume, I had to disable that, as it was messing up with my desired audio volume

**Comment by mapleroyal** [comment]
> Is this happening with AirPods? I had a similar experience with a podcast app, and it was due to the AirPod settings of Personalized Volume, I had to disable that, as it was messing up with my desired audio volume

No. I just checked, comparing the 16 pro max with my 15 pro max and pixel 9 pro xl—the problem is across all apps, not even just this audiobook app. Any sound that would exceed the loudness of 70% of the slider gets smashed down to a lower volume level. It makes the volume fluctuate constantly, because it's constantly trying to start at 100% (if that's where I've set the volume slider) and getting lowered rapidly by some software thing, over and over again any time talking starts (every few seconds in spoken content like audiobooks and podcasts).

**Comment by GianniCarlo** [comment]
Hmmm if it's happening across apps, looks like it's a system issue, does this also happen when playing through the iPhone speakers? Have you tried turning off and back on the phone? (I've seen audio peaking like this for example when playing audio through CarPlay and my SO goes and starts recording a story on Instagram, it sounds like the speakers want to blow up for a second, so not sure if something like this got stuck on your phone?)

**Comment by mapleroyal** [comment]
It happens exclusively through the phone speakers. It doesn't happen with any headphones, bt speaker, or car audio so far (thank god).

**Comment by moviebrain** [comment]
I understand this issue was opened awhile ago, but this also happens with an iPhone 17, and it is crazy on those moments when I’m listening to a book with the built in speaker. I’ve double verified that I’m not using any volume normalization, enabled normalization and re-disabled it. I don’t know what could be causing this but it is like a splinter I can’t quite free when it happens.

**Comment by moviebrain** [comment]
I poked around and tried to do some basic troubleshooting and am convinced the trouble isn’t specifically with BookPlayer. I can reproduce the volume/quality jumps within the same file at the same points, also same behavior in VLC for iOS. The fluctuations only happen with the speaker and not with any headphones, so it’s either a weird file issue (but why would that only affect the speaker?) or a bug with AVPlayer framework. Anyway, thanks for maintaining this app!

**Comment by moviebrain** [comment]
Ok, the common denominator for wonky volume from my speaker is a file format with files encoded with m4btool at 22050 sample rate and 64kbps. Every single one, re-encoded at a higher sample and bitrate from the original source and the behavior went away.
